### PR TITLE
feat: GEOT-192: group tags page

### PR DESCRIPTION
**Pull request by ra8ga**
;-), ;-), ;-)

### CODE REVIEW

**Comment by ra8ga** [comment]
retest this please

**Comment by ra8ga** [comment]
retest this please

File: packages/app-andrew/src/pages/group-tags/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { NextPage } from 'next'
+import styles from './styles.scss'
+import { useTranslation } from '@gtms/commons/i18n'
+// import { Button } from '@gtms/ui/Button'
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group-tags/index.tsx:5]: remove comment

File: packages/app-andrew/src/pages/group-tags/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { TagCard } from '@gtms/ui/TagCard'
+
+export const GroupTagsPage: NextPage<{}> = () => {
+  const { t } = useTranslation('groupTags')
+
+  return (
+    <div className={styles.wrapper} data-testid="group-tags-page">
+      <h2 className={styles.header}>{t('Tags')}</h2>
+      {/* <Button>Do stuff</Button> */}
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group-tags/index.tsx:14]: remote it

**Comment by ra8ga** [comment]
retest this please

File: packages/app-andrew/src/pages/group-tags/index.tsx (hunk 2 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group-tags/index.tsx]: removed

File: packages/app-andrew/src/pages/group-tags/index.tsx (hunk 1 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group-tags/index.tsx]: removed

**Comment by rfurmaga** [comment]
> from code perspective this looks ok, but what is that page? from where you want to take tag's image or description? there is no such functionality and it was never discussed. I dont see a point to merge it

That's what have had in previous version of the app https://ibb.co/FqT4hPP
Also it's in the doc geo groups light https://ibb.co/Gn5Q4h0
And here are some other draft screens of tags https://ibb.co/nMgzz9d

But anyway we can do it the same as we did it for the first time - hardcoded, so there will be tags and images that picked up and hardcoded in next iterations normal functionality can be added

**Comment by ra8ga** [comment]
> from code perspective this looks ok, but what is that page? from where you want to take tag's image or description? there is no such functionality and it was never discussed. I dont see a point to merge it

That's what have had in previous version of the app https://ibb.co/FqT4hPP
Also it's in the doc geo groups light https://ibb.co/Gn5Q4h0
And here are some other draft screens of tags that I've showed https://ibb.co/nMgzz9d

But anyway we can do it the same as we did it for the first time - hardcoded, so there will be tags and images that picked up and hardcoded in next iterations normal functionality can be added

**Pull request closed without merging**